search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -343,7 +355,8 @@
         issued-time (when-not (= issued-time (str umm-spec-date-util/parsed-default-date))
                       issued-time)
         modified-time (get-issued-modified-time update-time granule-end-date)
-        collection-citation (first collection-citations)]
+        collection-citation (first collection-citations)
+        browse-image-related-url (first (ru/browse-urls related-urls))]
```

Comment:
Minor:
Are we guaranteed that every browse URL will have all three of the the graphic preview fields? If not we might want to choose the URL to use based on how many fields they contain rather than just choosing the first one.

Reply:
No we are not guaranteed to have all three fields.

Reply:
ISO records won't have the mime-type.